connect/api/src/main/java/org/apache/kafka/connect/source/SourceConnector.java

```diff
@@ -24,4 +24,8 @@
  */
 public abstract class SourceConnector extends Connector {
 
+    @Override
+    public SourceConnectorContext context() {
```

Comment:
Would be great to have JavaDoc here as well, since it is changing the return type defined by the inherited method.